### ISSUE TITLE
Improving the interface or user experience

### ISSUE THREAD

**Issue by etechy** [adelyte/crescendo#84]
Though the 2.0 is tagged for medium residential projects, I find that even a mid project e.g. 25 light keypads or 16 video rooms require a two-step menu, like areas and rooms
Outside         >
First floor       > 
Second Floor > Master bedroom 
                          Office
                           Play room

Sorry if this has been approached before, but I find having this option is sometimes very important.
Any solution?

**Comment by DR7** [comment]
I recently had a customer ask about this as I had 17 audio zones and his wife said it was a rather large list to scroll and asked if it was possible to have the top left button in zones just as you describe. As an example his First floor has 5 audio zones, the Second floor has 4 audio zones, Garage has two audio zones and outside and decks make another 6 zones. I easily have that many light keypads but combined them into areas close together, I only have four video zones in this home.

**Comment by adelyte-austin** [comment]
We have done many proprietary implementations of `Zone/Device` lists nested by `Area` or `In_Use` status. Please email me privately to discuss your project's needs.

You can reduce list sizes by scoping in-wall touchpanels to nearby rooms and devices using the `\x01\x02\x03` ID syntax in the `Rooms-Sources Controller` and `Devices Controller` modules. The same can be done for personal mobile devices if rooms and devices are not necessary for all users.

**Comment by etechy** [comment]
Those are nice features of Crescendo. 
Outside of the elegance of this framework some workarounds are possible changing the interface, but I would not like to do it again.

**Comment by adelyte-austin** [comment]
I agree @etechy. Though one easy list scoping trick is to use an `Interlock` for the user to select the `Area` with the outputs manually linked to the `Subpage Reference List: Visibility` inputs. This produces and easy list sort. The downside is it requires manual configuration per panel up front, and if the order or length of the list changes.
